src/main/java/com/ratifire/devrate/util/zoom/webhook/service/ZoomWebhookService.java

```diff
@@ -36,16 +40,33 @@ public String handleZoomWebhook(String requestBody, Map<String, String> headers)
 
     String signature = headers.get("x-zm-signature");
     String timestamp = headers.get("x-zm-request-timestamp");
-
     zoomWebhookAuthService.validateSignature(signature, requestBody, timestamp);
 
     WebHookRequest payload = objectMapper.readValue(requestBody, WebHookRequest.class);
     String event = payload.getEvent();
-
     return switch (event) {
       case "endpoint.url_validation" -> zoomWebhookAuthService.handleUrlValidationEvent(payload);
-      case "meeting.ended" ->
-          interviewCompletionService.completeInterviewProcess(payload.getPayload().getMeeting());
+      case "meeting.ended" -> {
+        long meetingId = Long.parseLong(payload.getPayload().getMeeting().getId());
+
+        Interview interview = interviewService.getInterviewByMeetingId(meetingId);
+        ZonedDateTime scheduledStartTime = interview.getStartTime();
+        String endTime = payload.getPayload().getMeeting().getEndTime();
```

Comment:
Make the variable `payload.getPayload().getMeeting()` at the top to reduce duplicating